### PR TITLE
[SDK-2734] Automate releasing on CI

### PR DESCRIPTION
**Pull request by adamjmcgrath**
### Description

Use the orb with a couple of updates to accomodate monorepo

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -11,11 +12,25 @@ jobs:
       - node/install-packages
       - run:
           command: npm run build
+      - run:
```
Comment (lbalmaceda) [inline .circleci/config.yml:15]: if you are only going to use the "command" parameter for the `run` step, you can simplify these to:

`- run: npm run lint`

File: .circleci/config.yml [hunk 2]
```diff
@@ -11,11 +12,25 @@ jobs:
       - node/install-packages
```
Comment (lbalmaceda) [inline .circleci/config.yml:12]: this line can be replaced with `ship/node-install-packages` which relays the call to the circleci/node orb.

File: .circleci/config.yml [hunk 3]
```diff
@@ -1,6 +1,7 @@
 version: 2.1
 orbs:
   node: circleci/node@3.0.0
+  ship: auth0/ship@0.3.0
 jobs:
   test:
     executor:
```
Comment (lbalmaceda) [inline .circleci/config.yml:7]: this can make use of our `ship/node` executor, which by default uses image version `16.6.0`. Using those defaults you can remove the extra params:

```
executor: ship/node
```

If you want to change that to use specifically 15.x, you'd only need to change the param `name` to `ship/node` and keep the `tag` parameter the same. I just verified and that version has the minimum required **npm** version installed, as per this library's notice on top of the readme.

image

File: packages/examples/package.json [hunk 4]
```diff
@@ -7,6 +7,7 @@
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
   "scripts": {
+    "lint": "exit 0",
```
Comment (lbalmaceda) [inline packages/examples/package.json:10]: why is this needed for an example, what does it have to do with the main library under release?

**Review by lbalmaceda — CHANGES_REQUESTED** [review]
Found a few improvements that could make us remove the explicit dependency on the `circleci/node` orb, and a question around a changed script

File: packages/examples/package.json (hunk 4 above)
Comment (adamjmcgrath) [reply packages/examples/package.json:10]: When I edited the `prePublish` command in `packages/express-oauth2-jwt-bearer` to prevent the tests being run twice I moved the `lint` command to run on the whole monorepo which requires every package to have a `lint` script (or it fails)

**Review by lbalmaceda — APPROVED** [review]
